FactoryTest/FactoryTest.ino

```diff
@@ -3,16 +3,28 @@
 *
 * This is the main program for the v2 PAPR board Factory Test app.
 *
+* TODO:
```

Comment:
there is a specific directory structure which allows this.

Reply:
I deleted this, it should not have been there.

Reply:
That stuff shouldn't have been in the code. I have removed it.